src/models/multiChart.js

```diff
@@ -293,12 +292,11 @@ nv.models.multiChart = function() {
                     color: evt.point.color
                 };
                 tooltip
-                    .duration(100)
+                    .duration(0)
```

Comment:
why do you want duration 0 here?  Then it would jump instead of moving smoothly around between scatter points.